test/miner_txn_mgr_SUITE.erl

```diff
@@ -274,77 +274,6 @@ txn_invalid_nonce_test(Config) ->
 
     ok.
 
-txn_out_of_sequence_nonce_max_block_spantest(Config) ->
```

Comment:
how are these changes related to this PR?

Reply:
They were needed to boot blockchains for testing, otherwise totally unrelated

Reply:
so they were not used by txn_mgr test code?